### ISSUE TITLE
Install script fails upon first prompt

### ISSUE THREAD

**Issue by mleibner** [18F/laptop#144]
After running ```curl -s https://raw.githubusercontent.com/18F/laptop/master/laptop | sh -```, the build fails after the first prompt requiring user input (never allows me to answer the prompt - the script just exits).

### OS X version you are using
Sierra 10.12.2
### Did you recently upgrade from a previous OS X version?
No
### Are you running the script on a fresh OS X installation?
No
### Did you have any of the following tools installed before running the script?
- Xcode - Yes
- Apple's command line tools - Yes
- Anything the script installs (see "What it sets up" in the README) - No

### Copy and paste the entire contents of laptop.log, or attach it
```
Want to switch your shell from the default ^[[1mbash^[(B^[[m to ^[[1mzsh^[(B^[[m?
Both work fine for development, and ^[[1mzsh^[(B^[[m has some extra
features for customization and tab completion.
If you aren't sure or don't care, we recommend ^[[1mzsh^[(B^[[m.
Note that you can always switch back to ^[[1mbash^[(B^[[m if you change your mind.
Please see the README for instructions.
Press ^[[1my^[(B^[[m to switch to zsh, ^[[1mn^[(B^[[m to keep bash: failed

```

**Comment by monfresh** [comment]
Thanks for the report! I will look into this today.

**Comment by monfresh** [comment]
In the meantime, can you try this instead?
```
curl -s https://raw.githubusercontent.com/18F/laptop/master/laptop | bash -
```

**Comment by mleibner** [comment]
Thanks, @monfresh . I just tried that command with ```bash -```, but got the same error.

Haven't done any major customizations on this machine... here's my Terminal environment right now, in case it helps:
```
MacBook-Pro-1465:~ mleibner$ echo $0
-bash
MacBook-Pro-1465:~ mleibner$ bash --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin16)
Copyright (C) 2007 Free Software Foundation, Inc.
```

**Comment by monfresh** [comment]
@mleibner I think I found the solution. Can you try this please?
```
bash <(curl -s https://raw.githubusercontent.com/18F/laptop/master/laptop)
```

**Comment by mleibner** [comment]
Yes, that solution worked - thanks!

**Comment by mgwalker** [comment]
Closed by #145
